internal/pkg/cli/pipeline_init.go

```diff
@@ -66,13 +76,15 @@ type InitPipelineOpts struct {
 	// Caches environments
 	projectEnvs []string
 	fsUtils     *afero.Afero
+	buffer      bytes.Buffer
```

Comment:
Is it safe to reuse the same `bytes.Buffer` across multiple calls? It seems like we should be calling `bytes.Buffer.Reset` at least to clear the buffer between usages, or declare and use buffers closer to where we need them.

Reply:
The reason why we define buffer here basically is for the sake of testing. A good callout for reset!

Reply:
Addressed